src/together/utils/files.py

```diff
@@ -142,6 +142,18 @@ def _check_jsonl(file: Path) -> Dict[str, Any]:
                                 error_source="format",
                             )
 
+                        # Check that there are not extra columns
+                        for column in json_line:
+                            if (
+                                column
+                                not in JSONL_REQUIRED_COLUMNS_MAP[possible_format]
```

Comment:
This looks like a duplicate of L137, isn't it? Can you refactor?

Reply:
This line is required to filter extra columns that are not in other formats.